see https://bugs.gentoo.org/945000
use getopt from glibc/musl instead, getopt.h must be removed too
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -2,9 +2,9 @@ AM_CFLAGS = $(DCADEC_CFLAGS)
 AM_CPPFLAGS = -I$(top_srcdir)/include
 
 bin_PROGRAMS = dcadec extract_dca
-dcadec_SOURCES = dcadec.c getopt.h getopt.c gettimeofday.h gettimeofday.c
+dcadec_SOURCES = dcadec.c gettimeofday.h gettimeofday.c
 dcadec_LDADD = ../libdca/libdca.la ../libao/libao.la
-extract_dca_SOURCES = extract_dca.c getopt.c
+extract_dca_SOURCES = extract_dca.c
 
 man1_MANS = dcadec.1 extract_dca.1
 EXTRA_DIST = $(man1_MANS)
